README.md

```diff
@@ -120,7 +120,7 @@ DAPP架构请参考文章--[从架构维度看Web2.0与Web3.0应用之别](https
 33. [pooltogether](basic/33-pooltogether/README.md) ⌛
 34. [subgraph](basic/34-subgraph/readme.md) ⬜
 35. [Liquity](https://github.com/liquity) ⬜
-36. [NFT IPFS](basic/36-nft-ipfs/README.md) ✅
+36. [FileCoin](basic/36-nft-filecoin/README.md) ✅
```

Comment:
basic/36 is nothing related to filecoin. Why  rename it again to filecoin?

Reply:
task 10 has already finish nft-ipfs function.

Reply:
So it's duplicate one, right?

Reply:
So let's remove this one or replace it with another real filecoin task? @liberhe

Reply:
yes. This task is aim to research Filecoin.  Just replace it.

Reply:
> yes. This task is aim to research Filecoin. Just replace it.

Could you add a TODO for this task?

Reply:
o k